src/core/mount.c

```diff
@@ -2078,11 +2095,29 @@ static void mount_enumerate(Manager *m) {
                 }
 
                 (void) sd_event_source_set_description(m->mount_event_source, "mount-monitor-dispatch");
+
+                r = sd_event_add_post(m->event, &m->mount_enumerate_source, mount_enumerate_defer, m);
```

Comment:
Is it intended to use _add_post() here? If so, please rename the callback function.

Reply:
It's explained in commit message.
Nevertheless, renamed to less confusing  `late`.